traffic_ops/etc/init.d/traffic_ops

```diff
@@ -61,6 +61,7 @@ start ()
 {
 	stop
 	echo -e "Starting Traffic Ops\n"
+	ulimit -n 200000 || echo "Setting ulimit max files failed for traffic_ops_golang"
```

Comment:
This applies to both traffic_ops_golang and Perl, right? Would it be better to say "failed for Traffic Ops"?